org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/preferences/BuilderPreferenceAccess.java

```diff
@@ -56,6 +86,7 @@ public void initialize(IPreferenceStoreAccess preferenceStoreAccess) {
 
 		protected void initializeBuilderPreferences(IPreferenceStore store) {
 			store.setDefault(PREF_AUTO_BUILDING, true);
+			store.setDefault(PREF_SCHEDULING_RULE, SchedulingOption.WORKSPACE.name);
```

Comment:
am not sure if this will be affected by https://bugs.eclipse.org/bugs/show_bug.cgi?id=449010

Reply:
I'd go for the `SchedulingOption.name()`

Reply:
> am not sure if this will be affected by https://bugs.eclipse.org/bugs/show_bug.cgi?id=449010

Was not aware of that. Good point. For testing we should avoid the initialization then.